### DISCUSSION TITLE
针对本项目的一堆吐槽

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [share121/inter-knot#88] [其他]
# 使用 GitHub 作为服务：众所周知，GitHub 在中国地区的连通性……
# 你这 Console 里面是啥啊 
![image](https://github.com/user-attachments/assets/1123ade3-80b8-4a7e-8a58-a78d2926a620) 
# 调试代码往生产环境发我也是醉了
# 类似 #68 的，要是你的 GraphQL 访问 Token 到限制了，GitHub 直接拒绝你请求

**Comment by share121** [comment]
1. 选用 Github 是因为，我没有自己的服务器
2. 控制台的数据与界面上显示的内容一样，并没有什么用（现在已经修好了）
3. 我通过油猴脚本，让每个人用自己的 IP 访问 GitHub，所以他出问题，并不会影响整个网站
